packages/atomic/README.md

```diff
@@ -86,24 +50,41 @@ npm run cypress:test
 
 ## Utilities
 
-### The InitializeBindings & BindStateToController decorators
+### Stencil decorators
+
+To build custom Atomic components, it is highly recommended to use the same toolchain that we used, [Stencil](https://stenciljs.com/).
+We provide decorators which are used internally with most of our components.
 
-The `InitializeBindings` is an utility that automatically fetches the `bindings` from the parent `atomic-search-interface` component. This decorator should be applied to the `bindings` property directly.
+## InitializeBindings & BindStateToController decorators
 
-_Important_ In order for a component using this decorator to render properly, it should have an internal state bound to one of the property from `bindings`. This is possible by using either the `BindStateToController`decorator.
+`InitializeBindings` is an utility that automatically fetches the `bindings` from the parent `atomic-search-interface` or `atomic-external` component. This decorator has to be applied a property named `bindings`.
```

Comment:
`InitializeBindings is an utility` -> `InitializeBindings is a utility

`has to be applied a property named` -> `has to be applied to a property named`